src/charactersheet/models/dm/encounter.js

```diff
@@ -45,20 +45,6 @@ export class Encounter extends KOModel {
 //         }));
     };
 
-    /**
-     * Returns the list of encounter objects corresponding to the child ids.
-     */
-    getChildren = function() {
-//         var key = CoreManager.activeCore().uuid();
-//         return this.children().map(function(id, idx, _) {
-//             return PersistenceService.findByPredicates(Encounter, [
-//                 new KeyValuePredicate('encounterId', id),
-//                 new KeyValuePredicate('characterId', key)
-//             ])[0];
-//         });
-        return [];
-    };
-
     getParent = function() {
```

Comment:
It seems that the insides of this function is commented out. Should it just be removed?